packages/block-editor/src/components/block-fallback-web-version/style.native.scss

```diff
@@ -0,0 +1,56 @@
+.sheet__container {
```

Comment:
Nitpick: considering the BEM naming methodology, it feels odd to not name this merely `.sheet` IMO. 

Alternatively, given `sheet` is a reference to an implementation detail, I wonder if it makes more sense to name this `block-fallback` or similar.